### PR TITLE
PMM-11475 pmm2 dbaas upgrade tests with migration script

### PR DESCRIPTION
**Pull request by beatahandzelova**
docker way https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-upgrade-tests-beata-temp/detail/pmm2-dbaas-upgrade-tests-beata-temp/72/pipeline/
UI https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-upgrade-tests-beata-temp/detail/pmm2-dbaas-upgrade-tests-beata-temp/70/pipeline/

### CODE REVIEW

File: tests/verifyDBaaSUpgrade_test.js [hunk 1]
```diff
@@ -49,7 +69,7 @@ const pxcDbClusterDetails = new DataTable(['namespace', 'clusterName', 'node']);
 
 pxcDbClusterDetails.add(['default', `${pxc_cluster_name}`, '0']);
 
-Data(pxcDbClusterDetails).Scenario('PMM-T726 Verify PXC cluster monitoring after PMM Server upgrade @upgrade-dbaas',
+Data(pxcDbClusterDetails).Scenario('PMM-T726 Verify PXC cluster monitoring after PMM Server upgrade @upgrade-dbaas-after',
```
Comment (peterSirotnak) [inline tests/verifyDBaaSUpgrade_test.js:72]: Just a suggestion maybe better flag name would be @post-dbaas-upgrade. To be consistent.

File: tests/DbaaS/pages/dbaasActionsPage.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ module.exports = {
     }
   },
 
-  async deleteXtraDBCluster(dbClusterName, k8sClusterName, deleteCompleted = true) {
+  async deleteXtraDBCluster(dbClusterName, k8sClusterName, isDeleted = true) {
```
Comment (peterSirotnak) [inline tests/DbaaS/pages/dbaasActionsPage.js:79]: What is this boolean for? If false, then cluster will not be deleted? Do I understand it correctly?

File: tests/verifyDBaaSUpgrade_test.js [hunk 3]
```diff
@@ -10,32 +10,52 @@ Before(async ({ I }) => {
...
+  const { versionMinor } = homePage.getVersions();
 
   I.amOnPage(homePage.url);
   await homePage.upgradePMM(versionMinor, '', true);
   }
 );
 
-Scenario('PMM-T726 Verify DB clusters status and logs after PMM Server upgrade @upgrade-dbaas',
+// Scenario('Unregister Kubernetes cluster and register again @upgrade-dbaas-force-unregister',
+//   async ({ I, dbaasPage, dbaasAPI }) => {
```
[the remark this answers is not in the document]
Comment (beatahandzelova) [reply tests/verifyDBaaSUpgrade_test.js]: Done

File: tests/verifyDBaaSUpgrade_test.js (hunk 1 above)
Comment (beatahandzelova) [reply tests/verifyDBaaSUpgrade_test.js:73]: All tags go upgrade-dbaas-xxx seems pretty consistent to me.

File: tests/DbaaS/pages/dbaasActionsPage.js (hunk 2 above)
Comment (beatahandzelova) [reply tests/DbaaS/pages/dbaasActionsPage.js]: Removed, thanks.

**Comment by beatahandzelova** [comment]
New runs after resolving PR comments
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-upgrade-tests-beata-temp/detail/pmm2-dbaas-upgrade-tests-beata-temp/74/pipeline/
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-e2e-tests/detail/pmm2-dbaas-e2e-tests/2281/pipeline

**Review by peterSirotnak — APPROVED** [review]

**Pull request merged**